tables/tests/test_all.py

```diff
@@ -10,6 +11,9 @@
 from tables.tests.test_suite import suite, test
 
 
+faulthandler.enable()
```

Comment:
In principle this is OK for me, but I would prefer to use an environment variable if possible

Reply:
Ahh I had assumed that it would always be useful to have this enabled -- I've actually never encountered a case where I didn't want it enabled during testing. But I'll add support for something like `PYTABLES_ENABLE_FAULTHANDLER=0` that could be used, that way it's opt-out.